addons/mail/static/src/core/common/thread_model.js

```diff
@@ -248,9 +248,13 @@ export class Thread extends Record {
         return ["chat", "channel", "group"].includes(this.type);
     }
 
+    get typesAllowingCalls() {
+        return ["chat", "channel", "group"];
+    }
+
     get allowCalls() {
         return (
-            ["chat", "channel", "group"].includes(this.type) &&
+            this.typesAllowingCalls.includes(this.type) &&
```

Comment:
All types are allowing calls now, this is not really useful to keep a condition for that.

Reply:
Oh just whatsapp in enterprise maybe now, I forgot about it.

Reply:
So I guess fine to keep the condition.

Reply:
Yes whatsapp should be excluded, channel type conditions are a mess^^